### PR TITLE
Parse revision for PubIDs like NBS.LCIRC.145r11/1925

### PR DESCRIPTION
**Pull request by mico**
related #125

### CODE REVIEW

File: spec/nist_pubid/document_spec.rb [hunk 1]
```diff
@@ -925,6 +925,20 @@
...
+    context "NBS.LCIRC.118sup12/1926" do
+      let(:original_pubid) { "NBS.LCIRC.118sup12/1926" }
+      let(:short_pubid) { "NBS LC 118r1926sup12" }
+
+      it_behaves_like "converts pubid to different formats"
+    end
+
+    context "NBS.LCIRC.145r11/1925" do
+      let(:original_pubid) { "NBS.LCIRC.145r11/1925" }
+      let(:short_pubid) { "NBS LC 145r11-1925" }
```
Comment (ronaldtse) [inline spec/nist_pubid/document_spec.rb:937]: This is actually wrong -- in "r11/1925", the "11/1925" actually means 1925-November. So it should be "NBS LC 145/Upd-192511".

https://www.govinfo.gov/app/details/GOVPUB-C13-9f4bd1b0713ddbfadacc1a3190157cf3
Screenshot 2022-02-01 at 12 51 28 AM

File: spec/nist_pubid/document_spec.rb [hunk 2]
```diff
@@ -925,6 +925,20 @@
       it_behaves_like "converts pubid to different formats"
     end
 
+    context "NBS.LCIRC.118sup12/1926" do
+      let(:original_pubid) { "NBS.LCIRC.118sup12/1926" }
+      let(:short_pubid) { "NBS LC 118r1926sup12" }
```
Comment (ronaldtse) [inline spec/nist_pubid/document_spec.rb:930]: Similar to the document below, the "/" separates the month and the year.

https://www.govinfo.gov/app/details/GOVPUB-C13-0668d9f24b33835365f90efb62300b1a
Screenshot 2022-02-01 at 12 53 06 AM

This document should actually be: "NBS LC 118sup/Upd-192612"

**Review by ronaldtse — CHANGES_REQUESTED** [review]

**Comment by mico** [comment]
@ronaldtse Converted it to update

**Comment by ronaldtse** [comment]
Thanks @mico !

**Pull request merged**